backend/main.go

```diff
@@ -94,6 +94,8 @@ func run(ctx context.Context) int {
 	l.POST("/items", h.AddItem)
 	l.POST("/sell", h.Sell)
 	l.POST("/purchase/:itemID", h.Purchase)
+	l.GET("/:itemID/comments", h.GetCommentsByItemId)
+	l.POST("/:itemID/comments", h.AddComment)
```

Comment:
I think they should be `/items/:itemID/comments` like `/items/:itemID/image`

Reply:
@tonklable ok! changed!